ydb/core/tx/schemeshard/schemeshard_import.cpp

```diff
@@ -67,15 +108,17 @@ void TSchemeShard::FromXxportInfo(NKikimrImport::TImport& import, const TImportI
             import.SetProgress(Ydb::Import::ImportProgress::PROGRESS_PREPARING);
             break;
         case TImportInfo::EState::Transferring:
-            // TODO(ilnaz): fill items progress
-            import.SetProgress(Ydb::Import::ImportProgress::PROGRESS_TRANSFER_DATA);
+        case TImportInfo::EState::Done:
+            for (ui32 itemIdx : xrange(importInfo->Items.size())) {
+                FillItemProgress(this, importInfo, itemIdx, *import.AddItemsProgress());
+            }
+            import.SetProgress(importInfo->IsDone()
```

Comment:
Build is failed

```
/home/runner/actions_runner/_work/ydb/ydb/ydb/core/tx/schemeshard/schemeshard_import.cpp:115:44: error: no member named 'IsDone' in 'NKikimr::NSchemeShard::TImportInfo'
  115 |             import.SetProgress(importInfo->IsDone()
      |  
```

By the way, what difficulties do you encounter when trying to build it locally? Why it doesn't work for you?

Reply:
Let's just say that the computer that I have at work is not the best I've ever had. Most of the time while trying to build, it shuts down the terminal unfortunately. Thanks for the review.

Reply:
There are discord and telegram chat, you can find the links at the bottom of the following site:

https://ydb.tech/

Reply:
My eyes seen all of them except the Discord. Thanks a lot !